feature/isis/otg_tests/static_route_isis_redistribution_test/static_route_isis_redistribution_test.go

```diff
@@ -0,0 +1,718 @@
+	protoDst         = oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS
+)
+
+var (
+	advertisedIPv4 = ipAddr{address: dp2v4Route, prefix: dp2v4Prefix}
+	advertisedIPv6 = ipAddr{address: dp2v6Route, prefix: dp2v6Prefix}
+
+	dutPort1 = attrs.Attributes{
+		Desc:    "dutPort1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	atePort1 = attrs.Attributes{
+		Name:    "atePort1",
+		MAC:     "02:00:01:01:01:01",
+		IPv4:    "192.0.2.2",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:2",
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	dutPort2 = attrs.Attributes{
+		Desc:    "dutPort2",
+		IPv4:    "192.0.2.5",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:5",
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	atePort2 = attrs.Attributes{
+		Name:    "atePort2",
+		MAC:     "02:00:01:01:01:02",
+		IPv4:    "192.0.2.6",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:6",
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	dutPort3 = attrs.Attributes{
+		Desc:    "dutPort3",
+		IPv4:    "192.0.2.9",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:9",
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	atePort3 = attrs.Attributes{
+		Name:    "atePort3",
+		MAC:     "02:00:01:01:01:03",
+		IPv4:    "192.0.2.10",
+		IPv4Len: ipv4PrefixLen,
+		IPv6:    "2001:db8::192:0:2:a",
+		IPv6Len: ipv6PrefixLen,
+	}
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+type ipAddr struct {
+	address string
+	prefix  uint32
+}
+
+func verifyTraffic(t *testing.T, ate *ondatra.ATEDevice, flowsData []string) {
+
+	for _, flowName := range flowsData {
+
+		t.Logf("getting flow data for %v", flowName)
+		flow := gnmi.Get(t, ate.OTG(), gnmi.OC().Flow(flowName).Counters().State())
+
+		//Calculate the lower and upper bounds of the tolerance
+		lowerBound := float64(*flow.InPkts) * (1 - tolerancePercent)
+		upperBound := float64(*flow.InPkts) * (1 + tolerancePercent)
+
+		if float64(*flow.OutPkts) < lowerBound || float64(*flow.OutPkts) > upperBound {
+			t.Errorf("Failed: Flow %s due to Traffic TX %v Traffic RX %v tolerance range [%v - %v]",
+				flowName, *flow.InPkts, *flow.OutPkts, lowerBound, upperBound)
+		} else {
+			t.Logf("Passed: Flow %s Traffic TX %v Traffic RX %v within tolerance range [%v - %v]",
+				flowName, *flow.InPkts, *flow.OutPkts, lowerBound, upperBound)
+		}
+	}
+}
+
+type TableConnectionConfig struct {
+	ImportPolicy             []string `json:"import-policy"`
+	DisableMetricPropagation bool     `json:"disable-metric-propagation"`
+	DstProtocol              string   `json:"dst-protocol"`
+	AddressFamily            string   `json:"address-family"`
+	SrcProtocol              string   `json:"src-protocol"`
+}
+
+func getAndVerifyIsisImportPolicy(t *testing.T,
+	dut *ondatra.DUTDevice, DisableMetricValue bool,
+	RplName string, addressFamily string) {
+
+	gnmiClient := dut.RawAPIs().GNMI(t)
+	getResponse, err := gnmiClient.Get(context.Background(), &gpb.GetRequest{
```

Comment:
You can use ygnmi ops to use Get instead of Subscribe:

`gnmi.Get(t, dut.GNMIOpts().WithYGNMIOpts(ygnmi.WithUseGet())), .........)`

We would also need to update the hardcoded `DEFAULT` network instance with `deviations.DefaultNetworkInstance(dut)`

Reply:
tried to implement this but had hit the following errors. not sure if this needs to be structured in a different way.

path := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).TableConnection(
						oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC,
						oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS,
						oc.Types_ADDRESS_FAMILY_IPV4).State()

output := gnmi.Get(t, dut.GNMIOpts().WithYGNMIOpts(ygnmi.WithUseGet(),
						ygnmi.WithEncoding(gpb.Encoding_JSON_IETF)), path)
					t.Log(output)
					
noncompliant data encountered while unmarshalling: Noncompliance Errors by category (see https://github.com/openconfig/ygnmi#noncompliance-errors):
Path Noncompliance Errors: None
Type Noncompliance Errors:

&{DUT %!s(func(context.Context) (gnmi.GNMIClient, error)=0x10a02e7a0) map[] %!s(bool=true) <nil> [%!s(ygnmi.Option=0x106fa8ab0) %!s(ygnmi.Option=0x106fa8470)]} at /network-instances/network-instance[name=DEFAULT]/table-connections/table-connection[address-family=IPV4][dst-protocol=ISIS][src-protocol=STATIC]: path origin:"openconfig"  elem:{name:"network-instances"}  elem:{name:"network-instance"  key:{key:"name"  value:"DEFAULT"}}  elem:{name:"table-connections"}  elem:{name:"table-connection"  key:{key:"address-family"  value:"IPV4"}  key:{key:"dst-protocol"  value:"ISIS"}  key:{key:"src-protocol"  value:"STATIC"}}: value not present

Reply:
This is right. 

`https://github.com/openconfig/ygnmi#noncompliance-errors` non compliance errors are for logging only. The real issue here is `value not present`.

I see that in your previous implementation you used the `Config` path, however, in the new one you used `State`. The state value is not present and hence the error.

Reply:
Use the `Config` path instead?

Reply:
The problem with the `config` path is currently its not returning the `disable-metric-propagation` value so i would still end up needing to do this with the same approach as i have already implemented.